templates/forgot.html

```diff
@@ -26,16 +26,23 @@ <h3>{% trans %}Reset Password{% endtrans %}</h3>
                             <strong>{% trans %}Not found!{% endtrans %}</strong> {% trans %}We could not found any user with the given username.{% endtrans %}
                         </div>
                     </div>
-                {% endif %} 
+                {% endif %}
 
                 <form class="form form-horizontal" role="form" action= "/forgot" method="POST">
                     <div class="form-group">
                         <label for="username" class="control-label">{% trans %}Email Address{% endtrans %}</label>
-                        <input type="text" class="form-control" value="{{ username }}" name="username"/>
+                        <input
+                            autocomplete="email"
+                            class="form-control"
+                            id="username"
+                            name="username"
+                            type="email"
+                            value="{{ username }}"
+                        />
                     </div>
                     <div>
-                        <a class="btn btn-secondary" href="/{% if locale %}?locale={{ locale }}{% endif %}">{% trans %}Cancel{% endtrans %}</a>                
                         <input type="submit" class="btn btn-primary" value="{% trans %}Send password reset link{% endtrans %}"/>
+                        <a class="btn btn-secondary" href="/{% if locale %}?locale={{ locale }}{% endif %}">{% trans %}Cancel{% endtrans %}</a>
```

Comment:
In the rest of the gig-o, the order is "cancel" "action" - but the buttons are flush right to the dialog or whatever. Edit your profile and see the placement of the buttons at the bottom. So we should keep the order here but figure out how to make the flush right. (If you don't want to take this on, just revert this piece back and I'll add the layout as a bug)